firefox-devtools/profiler
# Move tooltip's lifecycle to react 16.3

# What is the purpose of the pull request

Update ```src/components/tooltip/Tooltip.js``` to use new lifecycle update from react 16.3
This fixes part of issue #1006 

# How to verify this PR

- Clone the branch
- cd into the directory
- yarn i
- yarn start
- Capture/Load a profile with firefox profiler
- Confirm the tooltip lays out correctly after switching from a wider tooltip to a smaller smaller one
